### ISSUE TITLE
RESPEAKER -2MIC HAT JST2.0 Speaker OUT

### ISSUE THREAD

**Issue by Sureshee** [respeaker/seeed-voicecard#39]
Hi,
I have a question regarding JST2.0 Speaker Out. Is it possible to connect a raw speaker directly on the Speaker Out terminals? or does it need any audio amplifier to connect with it. What is the output power on this Speaker out..? Is it 1W similar to Respeaker Core or is it different?
Looking forward to hear from you soon.
Regards
Suresh

**Comment by jabulonboaz** [comment]
Yes raw and the signal is already amplified i got a 3W 4ohm speaker on from Adafruit
